packages/os/bootstrap-containers@.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=bootstrap container %i
+Before=configured.target
+After=host-containerd.service
+
+[Service]
+Type=oneshot
+EnvironmentFile=/etc/bootstrap-containers/%i.env
+Environment=LOCAL_DIR=/local
+# Create directory for container persistent storage
```

Comment:
We're missing a `mkdir` here.

Reply:
Actually, that comment shouldn't be there. The persistent storage directory for the bootstrap container will be created by the `bootstrap-containers` binary, at the `preconfigured` stage. So when the bootstrap containers' systemd units are started on the `configured` target, the directories will be already there, that's why I omitted the `mkdir` as part of the unit.

Reply:
Can we clean up `host-containers@.service` as well, then?